lib/vtex/client.rb

```diff
@@ -21,7 +21,8 @@ def connection
 
     def self.resources
       {
-        products: ProductResource
+        products: ProductResource,
+        categories: CategoryAndBrandResource
```

Comment:
As we briefly discussed; it would be better to rename resources keys as singular

```
{
  product: ProductResource,
  category_and_brand: CategoryAndBrandResource,
}
```

but this change can be done within this PR or in another one ^^